solr/solrj/src/test/org/apache/solr/client/solrj/request/SolrPingTest.java

```diff
@@ -31,18 +30,18 @@ public class SolrPingTest extends EmbeddedSolrServerTestBase {
 
   @BeforeClass
   public static void beforeClass() throws Exception {
-    File testHome = createTempDir().toFile();
-    FileUtils.copyDirectory(getFile("solrj/solr"), testHome);
-    initCore("solrconfig.xml", "schema.xml", testHome.getAbsolutePath(), "collection1");
+    solrClientTestRule.startSolr(SolrTestCaseJ4.getFile("solrj/solr").toPath());
+
+    SolrTestCaseJ4.newRandomConfig();
+    solrClientTestRule.newCollection().withConfigSet("../collection1").create();
```

Comment:
do we need relative pathing like this?   I don't think that I see this normally...    Here is an example of the more common pattern: `solrClientTestRule.newCollection().withConfigSet(ExternalPaths.TECHPRODUCTS_CONFIGSET).create();`

Reply:
I concur that "I don't think that I see this normally".  But hey why not?  Your example looks nicer but it only applies to a well-known config.  Here in the SolrJ module, there are configSets in its own module that some tests use.  Would you propose creating a SolrJExternalPaths.COLLECTION1?  _Meh_.

Reply:
Fair...   I have a old PR open where I tried to deal with that issue of the solrj module...